docs/help-text.md

```diff
@@ -0,0 +1,151 @@
+# Other arguments
```

Comment:
How about naming this title and the file something like “CLI reference”?

Reply:
Changed!